synfig-studio/plugins/lottie-exporter/settings.py

```diff
@@ -71,12 +72,13 @@
 UNKNOWN_LAYER = "unknown_layer"
 CONVERT_METHODS = {"add", "atan2","average", "bone", "bone_link", "bone_root", "composite", "cos", "exp", "fromint", "linear", "radial_composite", "scale", "sine", "subtract", "switch", "vectorangle","weighted_average"}
 BONES = {"bone", "bone_root"}
-
+BLUR_TYPE = 29
 # Some waypoint animated definitions
 ANIMATED = 2
 SINGLE_WAYPOINT = 1
 NOT_ANIMATED = 0
-
+GROUP_FLAG = False #indicates whether layer is in a group or not
+LEVEL = 0 #Indicates the depth of a layer
```

Comment:
Just verifying that, did you take care of LEVEL for recursive group layers, right?
I mean, do you need a single variable LEVEL even if there are recursive group layers?

Reply:
Yes, single variable LEVEL works for recursive group layers as well. It is basically counting the number of layers that are above it in order.

Reply:
Alright, got it.